www/themes/unl/html/SiteMaster/Core/Registry/Site/EditForm.tpl.php

```diff
@@ -9,15 +9,17 @@
             <input type="email" id="support_email" name="support_email" multiple value="<?php echo $context->site->support_email ?>" />
         </li>
         <?php if ('unl' === $context->site->group_name): ?>
-        <li>
-            <label for="support_groups">mysupport.unl.edu assignments for this site (separated by spaces and quoted if the team name includes spaces).</label>
+          <li>
+            <label for="support_groups">UNL MySupport assignments for this site.</label>
```

Comment:
I probably should have mentioned that the "mysupport" name is going away. They have been referring to it as "Support" which is super unclear because there are more support avenues on campus that just I.T.  I would refer to it as "support.nebraska.edu" everywhere